src/plugins/janus_videoroom.c

```diff
@@ -2991,6 +2996,7 @@ static void janus_videoroom_create_dummy_publisher(janus_videoroom *room, GHashT
 /* Helpers for subscription streams */
 static janus_videoroom_subscriber_stream *janus_videoroom_subscriber_stream_add(janus_videoroom_subscriber *subscriber,
 		janus_videoroom_publisher_stream *ps,
+		const char *crossrefid,
```

Comment:
No need for a new line, just move after ps.

Reply:
fixed